xclim/indices/_threshold.py

```diff
@@ -2059,6 +2060,76 @@ def windy_days(
     return out
 
 
+@declare_units(ua="[speed]")
+def jetstream_metric_woolings(
+    ua: xarray.DataArray,
+) -> xarray.DataArray:
+    """Strength and latitude of jetstream.
+
+    Identify latitude and strength of maximum smoothed zonal windspeed in the region 15-75&deg;N, -60-0&deg;E.
+
+    Parameters
+    ----------
+    ua : xarray.DataArray
+      u-component wind velocity at between 750 and 950 hPa.
+
+    Returns
+    -------
+    xarray.DataArray
+      Daily time series of latitude of jetstream.
+    xarray.DataArray
+      Daily time series of strength of jetstream.
+
+    References
+    ----------
+    .. [woollings2010] Woollings, T., Hannachi, A., & Hoskins, B. (2010). Variability of the North Atlantic eddy‐driven jet stream. Quarterly Journal of the Royal Meteorological Society, 136(649), 856-868.
+
+    """
+
+    # select only relevant hPa levels, compute zonal mean windspeed
+    pmin = convert_units_to("750 hPa", ua.plev)
+    pmax = convert_units_to("950 hPa", ua.plev)
+
+    ua = ua.sel(plev=slice(pmin, pmax), lat=slice(15, 75), lon=slice(-60, 0))
```

Comment:
Same idea here with lat and lon. 
Also, should be check that the original grid includes the target region. And will this work if longitudes are defined from 0 to 360 ?

Reply:
I think using cf_array in this function is a good idea and would look something like:
```python
pmin = convert_units_to("750 hPa", ua.cf["air_pressure"])  
pmax = convert_units_to("950 hPa", ua.cf["air_pressure"])

ua = ua.cf.sel(air_pressure=slice(pmin, pmax), latitude=slice(15, 75), longitude=slice(lonmin, lonmax))
```
To determine `lonmin` & `lonmax` in above case, do you know if xclim or xarray already have any existing solution for checking whether longitude is -180-180 or 0-360? If not, I am happy to try and find or write something e.g. using 'lon % 180', etc.

Reply:
Not sure, the spatial subsetting logic originally in xclim has all been moved out to `clisops.subset`. I suggest here that we define the range of longitudes that is expected, and raise an error if the input does not meet that requirement, and possibly document how to change the coordinates in the docstring notes.